To improve compatibility with other rust packages, ease the chrono
version dependency to 0.4.

This indirectly reverts 9137b7fdccdb3e4a95ed7ae9b4652fbc6a1b4766 of upstream.

Upstream commented that the version dependency was made more strict,
because of a RUSTSEC in regards to chrono fix, which however doesn't affect
their (asn1) usage.

--- a/Cargo.toml.x	2022-09-07 08:15:26.234109943 +0000
+++ a/Cargo.toml	2022-09-07 08:15:35.585871922 +0000
@@ -25,7 +25,7 @@
 optional = true
 
 [dependencies.chrono]
-version = "0.4.20"
+version = "0.4"
 features = ["alloc"]
 default-features = false
 
